shared/util.go

```diff
@@ -1371,3 +1371,16 @@ func JoinTokenDecode(input string) (*api.ClusterMemberJoinToken, error) {
 
 	return &j, nil
 }
+
+// TargetDetect returns either target node or group based on the provided prefix:
+// An invocation with `target=h1` returns "h1", "" and
+// `target=@g1` returns "", "g1".
```

Comment:
just a nit, wont block merge, but we dont tend to have strict line length limits, so the comment breaking line here seems a bit odd. Not a problem though.

Reply:
I am used to break it in favor of split screen in the IDE. But I will adapt the repos code style :) Changed.

Reply:
I'll resolve this since it is not part of the PR anymore.